ee/modules/025-static-routing-manager/crds/doc-ru-routingtables.yaml

```diff
@@ -34,3 +34,5 @@ spec:
               nodeSelector:
                 description: |
                    Набор лейблов, определяющих узлы, на которых необходимо создать желаемые маршруты.
+
+                   > Значение должно быть допустимым значением метки: оно может содержать буквы (a-z, A-Z), цифры (0-9), тире (-), подчеркивания (_) и точки (.), должно начинаться и заканчиваться буквенно-цифровым символом (a-z, A-Z, 0-9) и не должно содержать «*».
```

Comment:
```suggestion
                   **Формат**: стандартный список `nodeSelector`.
```

Reply:
Let's simplify? We use this phrase in some other resources. [For example](https://deckhouse.io/products/kubernetes-platform/documentation/v1/modules/user-authn/cr.html#dexauthenticator-v1-spec-nodeselector)